src/View/User/Includes/_carousel.html.twig

```diff
@@ -7,13 +7,13 @@
         </div>
         <div class="carousel-inner">
             <div class="carousel-item active">
-                <img src="/assets/images/faceharas.jpg" class="d-block w-100">
+                <img src="/assets/images/{{ image1 }}" class="d-block w-100">
             </div>
             <div class="carousel-item">
-                <img src="/assets/images/poneycarousel.jpg" class="d-block w-100">
+                <img src="/assets/images/{{ image2 }}" class="d-block w-100">
             </div>
             <div class="carousel-item">
-                <img src="/assets/images/prairieharas.jpg" class="d-block w-100">
+                <img src="/assets/images/{{ image3 }}" class="d-block w-100">
```

Comment:
are images routes in database? => if so, you should have looped over all the image routes you got from db.
or are those the images that will always be in the doc? => if so, you could have left it the way it was

Reply:
We did not receive any images, so these are different images in the elements of a carousel which is an importable element. In the future, it will be possible to call up the image from a database.

Reply:
ok new code is coming :)